.github/workflows/nextjs.yml

```diff
@@ -0,0 +1,95 @@
+# Sample workflow for building and deploying a Next.js site to GitHub Pages
+#
+# To get started with Next.js see: https://nextjs.org/docs/getting-started
+#
+name: Deploy Next.js site to Pages
+
+on:
+  # Runs on pushes targeting the default branch
+  push:
+    branches: ['main']
```

Comment:
```suggestion
    branches: ['main']
pull_request:
    branches: ['main']
    
# Allows you to run this workflow manually from the Actions tab
  workflow_dispatch:    
```

Reply:
Sorry, the indentation is not correct :(

Reply:
No worries, just making sure that's how it was supposed to be 👍